### PR TITLE
[FEATURE] Add information about slot replacements and examples

### PR DESCRIPTION
**Pull request by linawolf**
Fix FQNs (Package not Mail)

See https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/10.3/Feature-90249-NewPSR-14EventsForExistingPackage-relatedSignalSlots.html

### CODE REVIEW

File: Documentation/ApiOverview/Events/Events/Core/Package/AfterPackageActivationEvent.rst [hunk 1]
```diff
@@ -6,9 +6,36 @@
...
+    The event was introduced to replace the Signal/Slot
+    `\TYPO3\CMS\Extensionmanager\Utility\InstallUtility::afterExtensionInstall`.
+
+The PSR-14 event :php:`\TYPO3\CMS\Core\Package\Event\AfterPackageActivationEvent`
 is triggered after a package has been activated.
 
+..  attention::
+    This event is only dispatched when an extension is activated in the
+    :guilabel:`Extension Manager`, therefore starting with TYPO3 v11 this
+    event is only dispatched in legacy installations, not in Composer-based
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Events/Events/Core/Package/AfterPackageActivationEvent.rst:19]: "event is only dispatched" used twice in the sentence. Maybe write something like

"This event is <del>only</del> dispatched when an extension is activated in the Extension Manager, therefore starting with TYPO3 v11 it is only fired in legacy installations, not in Composer-based ..."

Also for the other events.

File: Documentation/ApiOverview/Events/Events/Core/Package/_AfterPackageActivationEvent/_Services.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+services:
+  MyVendor\MyExtension\EventListener\MyEventListener:
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Events/Events/Core/Package/_AfterPackageActivationEvent/_Services.yaml:2]:
```suggestion
  MyVendor\MyExtension\Package\EventListener\MyEventListener:
```

File: Documentation/ApiOverview/Events/Events/Core/Package/AfterPackageActivationEvent.rst [hunk 3]
```diff
@@ -6,9 +6,36 @@
...
+
+..  literalinclude:: _AfterPackageActivationEvent/_Services.yaml
+    :language: yaml
+    :caption: EXT:my_extension/Configuration/Services.yaml
+
+An implementation of the event listener:
+
+..  literalinclude:: _AfterPackageActivationEvent/_MyEventListener.php
+    :language: php
+    :caption: EXT:my_extension/Classes/EventListener/MyEventListener.php
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Events/Events/Core/Package/AfterPackageActivationEvent.rst:37]:
```suggestion
    :caption: EXT:my_extension/Classes/Package/EventListener/MyEventListener.php
```

File: Documentation/ApiOverview/Events/Events/Core/Package/_AfterPackageActivationEvent/_MyEventListener.php [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace WCO\WcoLdap\EventListener;
+
+use TYPO3\CMS\Core\Package\Event\AfterPackageActivationEvent;
+
+class Setup
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Events/Events/Core/Package/_AfterPackageActivationEvent/_MyEventListener.php:7]:
```suggestion
final class MyEventListener
```

**Review by brotkrueml — CHANGES_REQUESTED** [review]

**Review by brotkrueml — COMMENTED** [review]
Argh, you overrode my commit which spells "Composer" with a capital C. So, please adjust it correctly again.

**Comment by linawolf** [comment]
Sorry about that, I corrected Composer to be a propper noun all over

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**